docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+
+Most beginner's confusion will be from distinguishing these different subnets. Note that only the C-chain is EVM-compatible and has Ethereum-compatible addresses. and most of dapps will be interacting with this chain.
+
+![Avalanche's diagram of subnets](https://docs.avax.network/assets/images/image(21)-3c5cb7f1f21926b05ae3631f453ed49d.png)
+
+## Setting up
+
+The quickest way to start is to run a group of simulator nodes locally. To do that, follow these steps:
+
+### Install and download
+
+- [Install Go](https://go.dev/dl/). Make sure to set `$GOPATH` variable to where you keep Go code (i.e. `$HOME/go`).
+
+- [Clone Avalanchego](https://github.com/ava-labs/avalanchego) (Avalanche node) and [Avalanch local simulator](https://github.com/ava-labs/ava-sim#readme). Make sure they are inside `$GOPATH/src/github.com/ava-labs`.
+
+- Make sure you have [Node.js](https://nodejs.org/en/) on your system to create the NFT app.
+
+### Run local simulator nodes
+
+- Build both projects with `./scripts/build.sh` included in the downloaded repositories.
+
+- In `ava-sim`, run the simulator with `./scripts/run.sh`, which relies on `avalanchego` being in the right place. The simulator runs a local network of 5 nodes listening on different ports. We will be using a node listening on port 9650.
+
+### Create a keystore user and add test fund
+
+Create a keystore user and store the credential on the target node (here, the node running on port 3650). Send a request to this API endpoint:
+
+```shell
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"keystore.createUser",
+    "params" :{
+        "username":"myUsername",
+        "password":"myPassword"
+    }
+}' -H 'Content-Type: application/json' 127.0.0.1:9650/ext/keystore
+```
+
+**Don't forget to replace "myUsername" and "myPassword" with your own.**
+
+> **Important**: You should only create a keystore user on a node that you operate, as the node operator has access to your plaintext password.
+
+Note the pre-funded "ewoq" private key, which is a test private key for getting some test AVAX in your account.
+
+```shell
+PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN
+```
+
+Import the private key to a C-chain address:
+
+```shell
+curl --location --request POST '127.0.0.1:9650/ext/bc/C/avax' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "method": "avax.importKey",
+    "params": {
+        "username":"username",
+        "password":"password",
+        "privateKey":"PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN"
+    },
+    "jsonrpc": "2.0",
+    "id": 1
+}'
+```
+
+Be thorough on the URL paths and the method namespace (again because of the subnets you might be running queries against the wrong one).
+
+Read more: [Funding a Local Network](https://docs.avax.network/build/tutorials/platform/fund-a-local-test-network)
+
+### Integrate with Metamask
+
+Set up Metamask to connect to a custom RPC address of the local network:
```

Comment:
Any website I can just click a button to do this for me? I think I've seen these around

Reply:
Do you mean a faucet?